apps/pie-storybook/utilities/index.ts

```diff
@@ -104,7 +104,8 @@ export const createVariantStory = <T>(
             return html`
         <div style="display: block; width: 100%;">
             ${propCombinations.map((props) => {
-                const typedProps = props as T & { darkBackground?: boolean };
+                const darkBackground = storyOpts?.bgColor === 'dark (container-dark)' || storyOpts?.bgColor === 'background-dark';
```

Comment:
Just a minor fix to ensure the background 👍🏼

Reply:
This issue then causes the background of the page to go dark as well as the container as seen in the Percy snapshots for both card and divider?

Reply:
Happy to change this back, however my justification for this change was that currently the dark background is applied to variant stories based on a `darkBackground` property that's included in the props object we pass into this function.

With this new approach, both `createStory` and `createVariantStory` set the background using the same implementation and removes the need to provide this property. IMO this new approach makes sense as `darkBackground` isn't a valid prop and will cause us issues when utilising our types as seen below:


![Screenshot 2025-01-14 at 16 11 18](https://github.com/user-attachments/assets/b7565f01-2d22-4b4d-857e-c34888c819bd)